dagshub/upload/wrapper.py

```diff
@@ -387,7 +387,7 @@ def __init__(self, repo: Repo, directory: str):
 
         """
 
-        self.files = {}
+        self.files: Dict[os.PathLike, Tuple[os.PathLike, BinaryIO]] = {}
         self.repo = repo
         self.directory = self._clean_directory_name(directory)
         self.request_url = self.repo.get_request_url(directory)
```

Comment:
not related to your change but self.request_url not used

Reply:
Thanks for noticing! Guess I'll get rid of it